sdk/tests/router/router_test.py

```diff
@@ -99,7 +99,7 @@ def test_create_router(turing_api, active_project, actual, expected, use_google_
     "actual,expected", [
         pytest.param(
             1,
-            turing.generated.models.InlineResponse200(id=1)
+            turing.generated.models.IdObject(id=1)
```

Comment:
These changes reflect the removal of the odd `InlineResponse` classes from the SDK code once the OpenAPI specs have been cleaned up in PR #170.

Reply:
im bit confused, was this `idobject` named as such for any reasons?

![image](https://user-images.githubusercontent.com/30390872/157361294-a544a24a-4ff8-48cb-93be-3f95f92e4b00.png)

Reply:
im bit confused, was this `idobject` named as such for any reasons?

![image](https://user-images.githubusercontent.com/30390872/157361294-a544a24a-4ff8-48cb-93be-3f95f92e4b00.png)

Reply:
`IdObject` is made to align with the return values of certain Turing API endpoints that return an object containing a field named `id`. 

`Id` on the other hand is made to align with the return values of *other* Turing API endpoints that return an integer representing some id.

If you're asking why Turing API was set up to return such integer values in a seemingly inconsistent way, we can perhaps bring this up again in some issue so that we can realign all the integer-related return values.